qiskit/circuit/library/__init__.py

```diff
@@ -394,98 +394,89 @@
 Techniques for the synthesis of reversible Toffoli networks, 2007
 http://dx.doi.org/10.1145/1278349.1278355
 
-.. autosummary::
-   :toctree: ../stubs/
-
-   templates.nct.template_nct_2a_1
-   templates.nct.template_nct_2a_2
-   templates.nct.template_nct_2a_3
-   templates.nct.template_nct_4a_1
-   templates.nct.template_nct_4a_2
-   templates.nct.template_nct_4a_3
-   templates.nct.template_nct_4b_1
-   templates.nct.template_nct_4b_2
-   templates.nct.template_nct_5a_1
-   templates.nct.template_nct_5a_2
-   templates.nct.template_nct_5a_3
-   templates.nct.template_nct_5a_4
-   templates.nct.template_nct_6a_1
-   templates.nct.template_nct_6a_2
-   templates.nct.template_nct_6a_3
-   templates.nct.template_nct_6a_4
-   templates.nct.template_nct_6b_1
-   templates.nct.template_nct_6b_2
-   templates.nct.template_nct_6c_1
-   templates.nct.template_nct_7a_1
-   templates.nct.template_nct_7b_1
-   templates.nct.template_nct_7c_1
-   templates.nct.template_nct_7d_1
-   templates.nct.template_nct_7e_1
-   templates.nct.template_nct_9a_1
-   templates.nct.template_nct_9c_1
-   templates.nct.template_nct_9c_2
-   templates.nct.template_nct_9c_3
-   templates.nct.template_nct_9c_4
-   templates.nct.template_nct_9c_5
-   templates.nct.template_nct_9c_6
-   templates.nct.template_nct_9c_7
-   templates.nct.template_nct_9c_8
-   templates.nct.template_nct_9c_9
-   templates.nct.template_nct_9c_10
-   templates.nct.template_nct_9c_11
-   templates.nct.template_nct_9c_12
-   templates.nct.template_nct_9d_1
-   templates.nct.template_nct_9d_2
-   templates.nct.template_nct_9d_3
-   templates.nct.template_nct_9d_4
-   templates.nct.template_nct_9d_5
-   templates.nct.template_nct_9d_6
-   templates.nct.template_nct_9d_7
-   templates.nct.template_nct_9d_8
-   templates.nct.template_nct_9d_9
-   templates.nct.template_nct_9d_10
+.. autofunction:: qiskit.circuit.library.templates.nct.template_nct_2a_1
```

Comment:
I don't know off the top of my head what the required import for these is, but judging by the import here, they probably also need putting in the correct Sphinx-documented module.